docs/release-notes.md

```diff
@@ -2,51 +2,34 @@
 # Release Notes
 
 If you haven't had a chance - checkout the [Letter From Devs](./letter.md) for a message from our development team.
-## Key Feature Changes for 1.3
-This section outlines key features changes between versions 1.2 and 1.3 for Open AMT Cloud Toolkit. 
+## Key Feature Changes for 1.4
+This section outlines key features changes between versions 1.3 and 1.4 for Open AMT Cloud Toolkit. 
 
 ### Additions
 #### RPS & MPS
-- **Stateless Authentication:** The RESTful APIs for both RPS and MPS had their APIKEYS removed and now leverage [Kong Gateway](https://konghq.com/kong/) for user authentication.  This simplifies how API calls are made into the Open AMT Cloud Toolkit services by routing all API connections through a single URL:   
-    Format:
-    ```html
-    https://example.url/
-    ```
-    Examples:
+- **Containerized Scaling Support:** In 1.2 we released a preview version of scaling.  This was a highly valuable exercise to learn how best to handle large scale deployments.  In 1.4 we are releasing our updated scaling capabilities with support for Docker Swarm and Kubernetes deployments.  This scaling feature will allow customers to scale out the MPS and RPS service as wide as needed to support their entire fleet of Intel&reg; AMT devices.  The following is a summary of some of the changes we have made to enable robust scaling:
```

Comment:
**Containerized Scaling Support:** In 1.2 we released a preview version of scaling, a first look at supporting large-scale deployments.  In 1.4 we've updated scaling capabilities with support for Docker Swarm and Kubernetes deployments.  
With this feature, scale the MPS and RPS microservices as needed to support an entire fleet of Intel&reg; AMT devices.  

The toolkit enables robust scaling through:

 * A new component (MPS Router) that acts as a reverse proxy between the API Gateway and the MPS.  More on MPS Router below.   
 *  Support for deploying cloud based components and dependencies to Docker Swarm and Kubernetes. Components include MPS, RPS, MPS Router, API Gateway, Vault, PostgreSQL.    
 * Cloud provider agnostic [deployment documentation](https://open-amt-cloud-toolkit.github.io/docs/1.4/Kubernetes/kubernetes/).     
 * Improved routing of KVM and SOL sessions when multiple instances of MPS are deployed.    
 * An increase in the number of  CIRA connections that can be handled per MPS instance.  Detailed guidance will be provided with the LTS release.<!--EndFragment-->



On that second bullet, I was really confused about the parens. I endeavored to remove the parens.

Reply:
Great changes!  implemented